app/src/main/java/com/example/scoutconcordia/ShuttleInfo.java

```diff
@@ -0,0 +1,587 @@
+            "10:20",
+            "10:35",
+            "11:10",
+            "11:30",
+            "11:45",
+            "12:20", "12:40", "12:55", "13:30", "14:05", "14:20", "14:40", "15:15", "15:30", "15:50", "16:25", "16:40", "17:00", "18:05", "18:40", "19:15", "19:50"};
+
+    String[] schedule_MonToThurs_SGW = {"07:45",
+            "08:05",
+            "08:20",
+            "08:35",
+            "08:55",
+            "09:10",
+            "09:30",
+            "09:45",
+            "10:05",
+            "10:20",
+            "10:55",
+            "11:10",
+            "11:45",
+            "12:00", "12:30", "13:00", "13:30", "14:00", "14:30", "15:00", "15:30", "16:00", "16:30", "17:00", "17:30", "18:00", "18:30", "19:00", "19:30",
+            "20:00",
+            "20:10",
+            "20:30",
+            "21:00",
+            "21:25",
+            "21:45",
+            "22:00",
+            "23:30",
+            "23:00"};
+
+    String[] schedule_friday_SGW = {
+            "07:45",
+            "08:20",
+            "08:55",
+            "09:30",
+            "09:45",
+            "10:05",
+            "10:55",
+            "11:10",
+            "11:45",
+            "12:05",
+            "12:20", "12:55", "13:30", "13:45", "14:05", "14:40", "14:55", "15:15", "15:50", "16:05", "16:25", "17:15", "18:05", "18:40", "19:15", "19:50"};
+
+
+    // This method checks the time and day and finds the next relevant shuttle bus time, from SGW.
+    // From there, it builds an estimate of how long it would take for the entire trip.
+
+    public double getNextEarliestTimeFromSGW() {
+
+
+        Calendar cal = Calendar.getInstance();
+//            cal.setTime(date);
+        double hour = cal.get(Calendar.HOUR_OF_DAY);
+        double minutes = cal.get(Calendar.MINUTE);
+
+        String timeString = (int) hour + ":" + (int) minutes;
+
+        // This variable keeps track of the current time.
+        double timeOfDay =  Math.round((hour + (minutes / 60)) * 100.0) / 100.0;
+
+        int currentDay = Calendar.getInstance().get(Calendar.DAY_OF_WEEK);
+
+        // This variable keeps track of the current day.
+        String today = null;
+
+        switch(currentDay){
+
+            case 1:
+                today = "Sunday";
+                break;
+
+            case 2:
+                today = "Monday";
+                break;
+
+            case 3:
+                today =  "Tuesday";
+                break;
+
+            case 4:
+                today =  "Wednesday";
+                break;
+
+            case 5:
+                today =  "Thursday";
+                break;
+
+            case 6:
+                today =   "Friday";
+                break;
+
+            case 7:
+                today =  "Saturday";
+                break;
+            default:
+                today = "Monday";
+                break;
+        }
+
+//            System.out.println(today);
+//            System.out.println("time is: " + timeOfDay);
```

Comment:
I am assuming this was for debugging purposes during development. 
Commented out code can be removed.

Reply:
Debugging comments have been removed.